tests/integration-tests/tests/networking/test_security_groups.py

```diff
@@ -14,21 +14,21 @@
 import boto3
 import pytest
 from assertpy import assert_that
-from cfn_stacks_factory import CfnStack
+from cfn_stacks_factory import CfnStack, CfnStacksFactory
 from remote_command_executor import RemoteCommandExecutor
 from troposphere import Ref, Template
 from troposphere.ec2 import SecurityGroup, SecurityGroupIngress
 from utils import check_head_node_security_group, create_hash_suffix, generate_stack_name, get_username_for_os
 
 
 @pytest.mark.usefixtures("os", "scheduler", "instance")
-def test_additional_sg_and_ssh_from(region, custom_security_group, pcluster_config_reader, clusters_factory):
+def test_additional_sg_and_ssh_from(region, custom_security_groups, pcluster_config_reader, clusters_factory):
     """
     Test when additional_sg ssh_from are provided in the config file
 
     The additional security group should be added to the head and compute nodes. The
     """
-    custom_security_group_id = custom_security_group.cfn_resources["SecurityGroupResource"]
+    custom_security_group_id = custom_security_groups
```

Comment:
`custom_security_groups` is always returning a list now, so this should be:
```python
custom_security_group_id = custom_security_groups[0]
```

Reply:
I just made the change!